src/components/RecipeDetails/RecipeDetails.tsx

```diff
@@ -0,0 +1,193 @@
+import { Link, useParams } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { getRecipe } from 'services/recipes';
+import { ListItem, List, ListItemText, Button } from '@material-ui/core';
+import './RecipeDetails.scss';
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Alert from '@material-ui/lab/Alert';
+
+const useStylesLoading = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      display: 'flex',
+      '& > * + *': {
+        marginLeft: theme.spacing(2),
+      },
+    },
+  })
+);
+const useStylesAlert = makeStyles((theme: Theme) =>
```

Comment:
You can create different fields in one styles object:

```suggestion
const useStyles = makeStyles((theme: Theme) => 
  createStyles({
    spinner: {
      width: '100%',
      '& > * + *': {
        marginTop: theme.spacing(2),
      },
    },
    anyOtherClass: {
      width: '100%',
      '& > * + *': {
        marginTop: theme.spacing(2),
      },
    },
  })
)
```

Reply:
Ok, i changed, everything is one field